x-pack/plugin/blob-cache/src/main/java/org/elasticsearch/blobcache/BlobCacheMetrics.java

```diff
@@ -20,19 +20,19 @@ public class BlobCacheMetrics {
     public BlobCacheMetrics(MeterRegistry meterRegistry) {
         this(
             meterRegistry.registerLongCounter(
-                "elasticsearch.blob_cache.miss_that_triggered_read",
+                "es.blob_cache.miss_that_triggered_read",
                 "The number of times there was a cache miss that triggered a read from the blob store",
                 "count"
             ),
             meterRegistry.registerLongCounter(
-                "elasticsearch.blob_cache.count_of_evicted_used_regions",
+                "es.blob_cache.count_of_evicted_used_regions",
```

Comment:
can we add a suffix `.total` to a metric name? I might change this later in https://github.com/elastic/elasticsearch/pull/103388 (under discussion)